docs/actions.md

```diff
@@ -1219,6 +1184,71 @@ This command starts a polling loop that continuously checks for logs from activa
 
   Similarly, whenever you run the poll utility, you see in real time the logs for any actions running on your behalf in OpenWhisk.
 
+## Getting actions
+
+Metadata that describes existing actions can be retrieved via the `wsk action get` command.
+
+```
+wsk action get hello
+ok: got action hello
+{
+    "namespace": "user@email.com",
+    "name": "hello",
+    "version": "0.0.1",
+    "exec": {
+        "kind": "nodejs:6",
+        "binary": false
+    },
+    "annotations": [
+        {
+            "key": "exec",
+            "value": "nodejs:6"
+        }
+    ],
+    "limits": {
+        "timeout": 60000,
+        "memory": 256,
+        "logs": 10
+    },
+    "publish": false
+}
+```
+
+### Getting an action URL
+
+An action can be invoked through the REST interface via an HTTPS request. To get an action URL, execute the following command:
```

Comment:
so this can be confusing... since the standard url is shown but if this is a web action it will be a different url and no auth may be required.

Reply:
the CLI doesn't offer the non-webaction url for a web action does it?

Reply:
@rabbah - the `action get --url` will return the web action (i.e. `/api/v1/web`) url for web actions, not the non-web action (i.e. `/api/v1/namespaces`) url.